install/kubernetes/helm/README.md

```diff
@@ -1,3 +1,9 @@
 # Installation using Helm
 
 Please follow the installation instructions from [istio.io](https://istio.io/docs/setup/kubernetes/install/helm/).
+
+# Development
+
+All development should take place in [istio-installer](https://github.com/istio-ecosystem/istio-installer). Istio 1.2 will be based on this new installer.
```

Comment:
I'd move the sentences around. 

```
Istio installation is based upon a new decomposed installation technology in Istio 1.2.  As such, all development work should occur in link.

If you have work that you feel should be backported to resolve defects, please submit to this repository so it can be added to the 1.1.z stream.
```